internal/domain/filter.go

```diff
@@ -551,7 +551,7 @@ func (f Filter) checkSizeFilter(r *Release, minSize string, maxSize string) bool
 		// string to bytes
 		minSizeBytes, err := humanize.ParseBytes(minSize)
 		if err != nil {
-			// log could not parse into bytes
+			r.addRejection("size: invalid minSize set")
```

Comment:
Shouldn't these return?

Reply:
Oopsie, you are probably right.

Reply:
Yeah, return false would be a good idea.

So maybe change to this:

```diff
-			r.addRejection("size: invalid minSize set")
+			r.addRejectionF("size: invalid minSize set: %s err: %q", minSize, err)
+			return false
```

And same for max size.